### ISSUE TITLE
How to set properties like `ScrollViewer.HorizontalScrollMode`?

### ISSUE THREAD

**Issue by VincentH-Net** [VincentH-Net/CSharpForMarkup#23]
### Should I call `Invoke` for setting simple properties like `ScrollViewer.HorizontalScrollMode`?

_Originally posted by @YegorStepanov in https://github.com/VincentH-Net/CSharpForMarkup/issues/19#issuecomment-1000878827_

**Comment by VincentH-Net** [comment]
@YegorStepanov it should be set like below, however it looks like a defect in the codegen prevents the `ScrollViewer_HorizontalScrollMode` method from being generated. I am fixing this now.

Explicit scrollviewer (works now):
```csharp
ScrollViewer (
    // ...
)  .HorizontalScrollMode().Enabled()
```

Implicit scrollviewer (needs fix):
```csharp
ListView (
    // ...
)  .ScrollViewer_HorizontalScrollMode().Enabled()
```

**Comment by VincentH-Net** [comment]
@YegorStepanov fixed in `0.6.1-dev.4`
